themes/classic/_dev/css/components/categories.scss

```diff
@@ -344,6 +344,33 @@
   }
 }
 
+#left-column,
+#right-column {
```

Comment:
Okay, you handle both columns, but what if we hook the module in another place? Have you checked the responsiveness of the module in all different part of the page?